monarchmoney/monarchmoney.py

```diff
@@ -1476,6 +1476,94 @@ async def get_transaction_category_groups(self) -> Dict[str, Any]:
             operation="ManageGetCategoryGroups", graphql_query=query
         )
 
+    async def create_transaction_category(
+        self,
+        group_id: str,
+        transaction_category_name: str,
+        rollover_start_month: datetime = datetime.today().replace(day=1),
+        icon: str = "❓",
```

Comment:
is this the best way to convey emojis?  should we use [Unicode character codes](https://home.unicode.org/emoji/) (e.g. `"\U00002753"`) or [CLDR description codes](https://cldr.unicode.org/index/charts) (e.g. `"\N{red question mark}"`).  It may no longer be the case, but some OSes have different visual representations (e.g. skin tones) than can cause for discrepancies between what you submit via API and what renders on Monarch.

Reply:
Thanks @hammem, I separated the PRs. And yes, good call out. I think it may be better to use the Unicode character codes as the default so I changed the default to `"\U00002753"`.